### DISCUSSION TITLE
Command alias change suggestions

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#95] [Ideas]
Suggestion: Swapping the term online for "who", swapping the term status for "info". These are most commonly used for these things.

**Comment by Volte6** [comment]
`status` or `stat` has always been correct for me, but I can add aliases for info, and make `who` show who's online, sure.

**Reply by ArmoredArtist** [reply]
Perfect solution!

**Comment by Volte6** [comment]
Addressed in https://github.com/Volte6/GoMud/pull/101

Closing.
